factcast-store/src/test/java/org/factcast/store/internal/PgSubscriptionFactoryTest.java

```diff
@@ -18,21 +18,24 @@
 import static org.mockito.Mockito.*;
```

Comment:
Using mockito static magic, we can also test if the ES is actually used, no?

Reply:
done :)